### PR TITLE
[incubator/argo-ingress] Import new chart for CloudFlare Ingress Controller

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add the Argo, the `cloudflare-ingress-controller` (aka `argo-ingress`)
* Make target `update` for easily updating from remote repo

## why
* Expose kubernetes services without needing to expose anything publically
* Alternative to bitly-oauth2 proxy (which appears unmaintained and is definitely buggy!)
* BeyondCorp-style security model'
* Stackpoint chart is out-of-date (`0.5.0`); latest is `0.5.1`

## demo

image

## references
* https://github.com/StackPointCloud/trusted-charts/blob/master/stable/argo-ingress/Chart.yaml#L3
* https://github.com/cloudflare/cloudflare-ingress-controller

### CODE REVIEW

**Comment by osterman** [comment]
Wanted by https://github.com/cloudposse/helmfiles/pull/17/files

**Comment by osterman** [comment]
Rename the chart to `argo-ingress` since that's what the "official" one is called, despite the repo name being `cloudflare-ingress-controller`

File: incubator/argo-ingress/values.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Cloudflare account details
+
+# The secret contains the base64-encoded origin certificate for your cloudflare domain.
+# Set the install flag false to configure manually in the cluster later.
+# See https://developers.cloudflare.com/argo-tunnel/quickstart/quickstart/ for information on
+# obtaining the origin certificate
+secret:
+  install: false
+  domain: example.co
+  certificate_b64: LS0tLS1CRUdJT...insert base64-encoded certificate here
```
Comment (goruha) [inline incubator/argo-ingress/values.yaml]: Should be empty by default

Comment (osterman) [reply incubator/argo-ingress/values.yaml]: Conflicted on whether or not it makes sense to update. The source of the problem is upstream.

Comment (osterman) [reply incubator/argo-ingress/values.yaml]: (we just clone the repo from cloudflare)

**Comment by osterman** [comment]
I think we can close this now. Cloudflare has released their own managed chart repo.

https://github.com/cloudflare/helm-charts

```
helm repo add cloudflare https://cloudflare.github.io/helm-charts
helm repo update
```

**Pull request closed without merging**